src/components/MenuXss.vue

```diff
@@ -141,6 +141,10 @@
           />
         </VList>
       </VMenu>
+      <VListItem
```

Comment:
Oh, I just notice that this was added to the XSS menu.

Why isn't it added to the Encoding menu? The function name seems to be related to Encoding as well.

Reply:
I added it to the XSS menu because I believe the primary, if not the only, use case for this helper function will be XSS.
We probably won't need to use this anywhere else I guess?

Reply:
You are right.